sliceUtility/sliceUtil.go

```diff
@@ -0,0 +1,38 @@
+package slice
+
+func IsPresent(s []string, e string) bool {
```

Comment:
1. Use name return for small functions
2. We use an elimination pattern in Go which reduce the nesting of the code block
`
for _, val := range s {
		if  val != e {
			continue
		}
		return true
	}
`